components/KeywordLink.js

```diff
@@ -5,7 +5,7 @@ import HashIcon from '../public/img/hashicon.svg';
 export const KeywordLink = ({ children }) => (
   <div className="block py-1 text-accent-500 md:inline md:px-5 md:first:pl-0">
     <HashIcon className="mr-3 -mt-1 inline w-3" />
-    <Link href="/">
+    <Link href={`/keyword/${children}`}>
       <a className="underline hover:text-black hover:no-underline">
         {children}
       </a>
```

Comment:
It should be URL encoded, so if the user will type some special character, it won't break it.

In reality, if you would use the user's input functionally it MUST be escaped/sanitized to avoid XSS